css/index.css

```diff
@@ -1,3 +1,390 @@
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+body {
+	max-width: 1200px;
+	margin: 0 auto;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+header {
+	display: flex;
+	justify-content: flex-start;
+	flex-wrap: nowrap;
+	align-items: flex-start;
+
+	
+
+
+
+
+}
+header hr {
+	display: none;
+}
+nav {
+	width: 90%;
+	display:flex;
+	justify-content: space-around;
+	padding-top: 10px;
+	text-decoration: none;
+}
+nav a{
+	text-decoration: none;
+	color: black;
+}
+
+.top-sec {
+
+	 display:flex;
+	 justify-content: space-around;
+	 flex-wrap: wrap;
+
+
+}
+
+.top-sec img {
+
+	display: flex;
+	padding-top: 45px;
+	width: 45%;
+	height: 45%;
+}
+
+.cta {
+
+	display: flex ;
+	justify-content:  center;
+	flex-wrap: wrap;
+	text-align: center;
+
+
+
+
+}
+
+.cta h1 {
+
+	display: flex;
+	font-size: 6rem;
+	padding-top: 1em;
+
+}
+
+.cta button {
+
+
+	 background-color: white;
+	 color: black;
+	 width: 60%;
+	 height: 2rem;
+
+}
+
+.middle-sec {
+
+	display:flex;
+	padding-top: 1em;
+	justify-content: space-between;
+
+
+
+}
+.middle-sec .features {
+
+	display: flex;
+	flex-direction: column;
+  padding-left: 1em;
+	width: 35%;
+
+
+}
+.middle-sec .about {
+
+display: flex;
+flex-direction: column;
+padding-right: 1em;
+width: 35%;
+
+}
+.bottom-img {
+
+	 display:flex;
+	 justify-content: space-around;
+	 width: 100%;
+
+}
+.bottom-img img {
+
+width: 100%;
+}
+
+.bottom-sec {
+	display: flex;
+	justify-content: space-between;
+	padding-top: 1em;
+
+}
+.bottom-sec .services {
+
+	display: flex;
+	width: 25%;
+	flex-direction: column;
+	padding-left: 1em;
+
+}
+
+.bottom-sec .product {
+
+display: flex;
+width: 25%;
+flex-direction: column;
+
+}
+
+.bottom-sec .vision  {
+
+	display: flex;
+	width: 25%;
+	flex-direction: column;
+	padding-right: 1em;
+
+
+
+
+}
+
+.contact h4 {
+	padding-top: 10px;
+
+}
+
+
+
+footer {
+
+	text-align: center;
+
+}
+
+
+
+
+@media (max-width: 1024px) {
+
+	body {
+		max-width: 1024px;
+		margin: 20px;
+	
+	}
+	header hr {
+		display: none;
+	}
+
+.cta {
+	display: flex;
+	flex-wrap: nowrap;
+	flex-direction: column;
+
+
+}
+.cta button {
+	padding-left: 20px;
+	margin-left: 64px;
+		margin-top: 20px;
+		width:46%;
+}
+.cta h1 {
+	font-size: 70px;
+}
+.Contact {
+	padding-top: 10px;
+	padding-left: 10px;
+}
+}
+	
+
+@media (max-width: 800px) {
+
+	body {
+		max-width: 800px;
+		max-height: 1370px;
```

Comment:
```suggestion

```

Reply:
Typically you won't need to set a height, most styling methods will result in the height being set by what space the content needs to take up